config/packages/security.yaml

```diff
@@ -140,12 +150,3 @@ security:
         - { path: "%sylius.security.admin_regex%", role: ROLE_ADMINISTRATION_ACCESS }
         - { path: "%sylius.security.api_regex%/.*", role: ROLE_API_ACCESS }
         - { path: "%sylius.security.shop_regex%/account", role: ROLE_USER }
-
-        - { path: "%sylius.security.new_api_admin_regex%/authentication-token", role: IS_AUTHENTICATED_ANONYMOUSLY }
```

Comment:
I believe we need some protection for the `/new-api` URLs and no protection for authentication endpoints here as well. Could you add them in a separate PR?